Overview.bs

```diff
@@ -601,7 +700,7 @@ the new value for that codepoint.
 A server is free to choose any codepoint ordering, but should try to pick one that will minimize the
```

Comment:
General comment: one thing I noticed is that the current version of the spec seem to be written as a document that describes a particular PFE implementation. The alternative way is to describe what implementations [must / must not / should / should not] do, and all these requirement need to be clearly identified as normative (and, therefore, testable).
If something is expected to be done a particular way (e.g. checksum calculation below), it needs to be defined as normative - the difference is whether something can be done this way or MUST be done this way.